### PR TITLE
Update Mac building instructions.

### PR DESCRIPTION
**Pull request by chrisseaton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ On the Mac you can use Homebrew:
 
     brew tap homebrew/versions
     brew install gcc46 --with-fortran
-    brew install gmp4
+    brew link --force gmp4
```
Comment (woess) [inline README.md]: don't you still have to install it first?

Comment (chrisseaton) [inline README.md]: No, it's a dependency of `gcc46` - I didn't realise before.

Comment (woess) [inline README.md]: ah, ok!

**Comment by woess** [comment]
I have the suspicion that it's a bug of the gcc46 package that the fortran library is not linked...

**Pull request merged**